test/e2e.v3.test.ts

```diff
@@ -167,9 +167,11 @@ describe("v3 E2E Test Scenarios", () => {
       const newDocument = await wrapDocument(datum[2], { version: SchemaId.v3 });
       const obfuscatedDocument = await obfuscate(newDocument, ["key2"]);
 
-      const verified = verifySignature(obfuscatedDocument);
-      expect(verified).toBe(true);
-      expect(validateSchema(obfuscatedDocument)).toBe(true);
+      const signatureValidity = verifySignature(obfuscatedDocument);
```

Comment:
I find it useless to create a temporary var for this, can directly call the function in the expect

Reply:
fixed!